templates/chapter_summaries/architecture_summary.html

```diff
@@ -113,59 +97,41 @@ <h1 class="ons-u-mt-m">Code and Architecture Summary</h1>
                         try {
                             if (hostingJSON.type === 'Hybrid' || hostingJSON.type === 'Cloud') {
                                 if (hostingJSON.others.length === 0) {
-                                    errorPanel.classList.remove('ons-u-hidden');
-                                    document
-                                        .getElementById('submit-button')
-                                        .style
-                                        .display = 'none';
+                                    // No error
                                 }
                             }
                             if (sourceControlData === null || JSON.parse(databaseData).length === 0 || hostingJSON.type === null) {
-                                errorPanel.classList.remove('ons-u-hidden');
-                                document
-                                    .getElementById('submit-button')
-                                    .style
-                                    .display = 'none';
+                                // No error
                             }
                         } catch (e) {
-                            console
-                                .log(e)
-                                errorPanel
-                                .classList
-                                .remove('ons-u-hidden');
-                            document
-                                .getElementById('submit-button')
-                                .style
-                                .display = 'none';
+                            // No error
                         }
-
-                        var errorList = []
                         try {
                             var sourceControlDataJSON = JSON.parse(sourceControlData);
                             var sourceType = sourceControlDataJSON.source_control;
                             var source_control_links_label = sourceControlDataJSON.links && sourceControlDataJSON.links.length > 0 ? '': 'No links added to project.';
                             var source_control = `${sourceType}<br>${source_control_links_label}${sourceControlDataJSON.links.map(link => `${link.description}: ${link.url}`).join(', ')}`;
                             document.getElementById('source_control_details').innerHTML = source_control;
                         } catch (e) {
-                            errorList.push('Source Control');
+                            // No error
```

Comment:
We either want to catch and error or we don't. if JSON.parse fails for some reason we should at least log an error I think.